### PR TITLE
Adding case DIVD-2022-00014

### PR DESCRIPTION
**Pull request by MrSeccubus**

### CODE REVIEW

File: csirt.divd.nl/_cases/DIVD-2022-00014.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+layout: case
+title: "GrayNoise's Ukraine only list"
```
Comment (cookiemonster) [inline csirt.divd.nl/_cases/DIVD-2022-00014.md:3]: GreyNoise's

File: csirt.divd.nl/_cases/DIVD-2022-00014.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+excerpt: "GreyNoise has created a public list of IP addresses that have exclusively been observed in their honeypots in the Ukraine, and not anywhere else.
```
Comment (cookiemonster) [inline csirt.divd.nl/_cases/DIVD-2022-00014.md:4]: in Ukraine,

File: csirt.divd.nl/_cases/DIVD-2022-00014.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+We decided to take it upon ourselves to make network administrators aware of the fac that these hosts are on this list."
```
Comment (cookiemonster) [inline csirt.divd.nl/_cases/DIVD-2022-00014.md:5]: remove: of the fact

File: csirt.divd.nl/_cases/DIVD-2022-00014.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  event: "Case opened"
+- start: 2022-03-04
+  end:
+  event: "First notifications sent"
+ips: 0 
+# You can set IPs to n/a when this case isn't about IPs (e.g. stolen credentials)
+---
+## Summary
+
+[GreyNoise](https://greynoise.io/) has published a "free, public, unauthenticated, self-updating feed of all IPs that are exclusively targeting devices geographically located in Ukraine's IP space with scans, exploits, etc." We first can aware of it via a [tweet](https://twitter.com/Andrew___Morris/status/1496923547872206852?s=20&t=Y5vBKAso8TRrwief4KpbLg) form GreyNoise founder Andrew Morris. We feel that in these times network administrators should be aware of these IP addresses even if they are unaware of the services of GreyNoise.
```
Comment (cookiemonster) [inline csirt.divd.nl/_cases/DIVD-2022-00014.md:37]: "We first became aware"

form --> form

 We feel that in these times, network administrators

File: csirt.divd.nl/_cases/DIVD-2022-00014.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+We are using the list of "unspoofed" IPs (IP addresses that have completed a full three way handshake) located on [GreyNoise Pulse](https://www.greynoise.io/viz/pulse). If you want to check other lists, feel free to inspect the [manifest](https://api.greynoise.io/datashots/ukraine/manifest.json) of the full API.
+
+## What you can do
+
+We are sending out these emails for awareness only. We want you to be aware of the fact that you have one or more nodes in your network that are displaying this behaviour. How you act on this information is completely up to you. We cannot determine if this behaviour is expected, unexpected, logical, illogical, wanted or unwanted by you.
```
Comment (cookiemonster) [inline csirt.divd.nl/_cases/DIVD-2022-00014.md:43]: We are sending out these emails for awareness only. We want you to be aware that you have one or more nodes in your network displaying this behavior. How you act on this information is entirely up to you. We cannot determine if this behavior is expected, unexpected, logical, illogical, wanted, or unwanted by you.

File: csirt.divd.nl/_cases/DIVD-2022-00014.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+## What we are doing
+
+On a regular basis we will retreive the list from GreyNoise and sent notifications to network administrators. We will only sent out notifications for systems last seen on or after 1 March 2022 and only to system we have not previously sent a notification about.
```
Comment (cookiemonster) [inline csirt.divd.nl/_cases/DIVD-2022-00014.md:47]: On a regular basis, we will retrieve the list from GreyNoise and send notifications to network administrators. We will only send out notifications for systems last seen on or after 1 March 2022 and only to system owners; we have not received a notification before.

**Review by cookiemonster — CHANGES_REQUESTED** [review]
Need only a few minor changes.

File: csirt.divd.nl/_cases/DIVD-2022-00014.md (hunk 2 above)
Comment (MrSeccubus) [reply csirt.divd.nl/_cases/DIVD-2022-00014.md]: Stom, dit wist ik en ik let er al de hele week op.

**Pull request merged**